django_cloud_deploy/tool_requirements.py

```diff
@@ -263,25 +263,33 @@ def handle(cls, console: io.IO):
             process.close()
 
 
-_REQUIREMENTS = [
-    Gcloud,
-    Docker,
-    CloudSqlProxy
-]
-
-
-def check_and_handle_requirements(console: io.IO):
+_REQUIREMENTS = {
+    'gke': [
+        Gcloud,
+        Docker,
+        CloudSqlProxy
+    ],
+    'gae': [
+        Gcloud,
+        CloudSqlProxy
+    ]
+}
+
+
+def check_and_handle_requirements(console: io.IO, backend: str):
     """Checks that requirements are installed. Attempts to install missing ones.
 
     Args:
         console: IO, class that handles the input/output with the user.
+        backend: str, Defines which platform on Determines what requirements are
```

Comment:
Why "Determines" is capitalized?

Also, maybe we do not need to put type information here. It shows as type annotation for parameters.